products/platform/documentation-site/research/search-design-doc.md

```diff
@@ -1,15 +1,159 @@
-# Global Documentation Search
+# Documentation Search MVP
 
-Author: Bill Fienberg
-Last Updated: 2020-01-23
-Reviewers: TBD
+**Author(s):** Bill Fienberg <bill.fienberg@oddball.io>
 
-## Objective
+**Last Updated:** 2020-02-04
 
-## Background
+**Status:** **Draft**
+
+**Approvers:** Andrew G., Megan K., Rian F., VSP Ops
```

Comment:
[meta] "VSP Ops" isn't a person, find an individual who you think should approve